data-cube-curation/values.yaml

```diff
@@ -34,7 +34,7 @@ replicaCount: 1
 
 image:
   repository: docker.io/zazuko/datacube
-  tag: api_0.2.0
+  tag: "{{ .Chart.AppVersion }}"
```

Comment:
The one downside of this change would be that Renovate (a bot that automatically creates new PRs if there are newer docker tags available) will stop working for this chart, thus stop notifying us about new releases, because I don't think it can parse the `{{ .Chart.AppVersion }}` expression as a semver number.

Personally I'm against that change and accept a little duplication in that case. But I don't have a hard feeling either for this chart :)

Reply:
WDYT @srueg ?

Reply:
Relevant documentation: https://docs.renovatebot.com/modules/manager/helm-values/

Reply:
Hmm good point. I'd also recommend letting Renovate do it's job.
Since this is mostly managed by Zazuko it's up to you @sandhose

Reply:
Well, if we let renovate update the image tag, it means the appVersion field will be out of sync (as renovate does not seem to update it) ; not that it's very important, but I think I prefere updating the appVersion manually every once in a while

Reply:
I understand you point. However, when Renovate creates a PR with the new image tag, we have to touch the PR in any case, as Renovate does not update `version` in `chart.yaml`. A change in the helm chart without updating the chart version will fail the pipeline so we cannot merge it either way. Maybe that changes in the future when Renovate supports that, but for now it's "okay", at least we get notified automatically.

We can accept your change, no problem. But then you would have to create PRs on your own if the chart should be updated with newer image tags, as we don't check for new releases every day :)

Reply:
> We can accept your change, no problem. But then you would have to create PRs on your own if the chart should be updated with newer image tags, as we don't check for new releases every day :)

Sounds good to me 👍